Code/GraphMol/Fingerprints/Fingerprints.cpp

```diff
@@ -173,6 +173,182 @@ return res;
 
 }  // end of anonymous namespace
 
+
+namespace utils {
+
+void buildDefaultAtomInvariants(const ROMol &mol,
```

Comment:
Maybe rename to `buildDefaultRDKitFingerprintAtomInvariants` just to make this more explicit?